comptest/web/management/commands/evaluator.py

```diff
@@ -176,19 +177,31 @@ async def ahandle(self):
                 e = Evaluation(version=v)
                 await e.asave()
 
-            # Start Evaluations when they have not been started yet
+            # Get the Evaluations that have not been started yet
             unstarted_evaluations = Evaluation.objects.select_related("version").filter(
                 status=Evaluation.Status.NOT_STARTED
             )
 
-            async for e in unstarted_evaluations:
-                await self.start_evaluation(evaluator, e)
-
-            # Check running evaluations
+            # Get the running Evaluation objects
             running_evaluations = Evaluation.objects.select_related("version").filter(
                 status=Evaluation.Status.EVALUATING
             )
 
+            # Get the number of running evaluations
+            # This is a synchronous operation so it's wrapped in sync_to_async
+            num = await sync_to_async(running_evaluations.count)()
```

Comment:
There's async versions of a lot of these! For `count`, there's `acount` (https://docs.djangoproject.com/en/5.1/ref/models/querysets/#django.db.models.query.QuerySet.acount) which makes this async. Switch to using that so we can remove the sync_to_async?